Operators/Types/lib/img/generate/RoundedRect_bf7fcd04-0cf6-4518-86cc-48f74483d98d.t3ui

```diff
@@ -1,49 +1,55 @@
 {
   "Id": "bf7fcd04-0cf6-4518-86cc-48f74483d98d"/*RoundedRect*/,
-  "Description": "",
+  "Description": "Generates a rounded rectangle.\n\nCan be customised to create a variety of shapes, similarly to [Blob].\n\nAKA: rectangle, square, ellipse, circle, rings, blob, vignette\n\nTips:",
   "InputUis": [
     {
       "InputId": "6b6887d4-ebaa-4489-b993-5027789ce1c7"/*Image*/,
       "Position": {
         "X": -195.9,
         "Y": -128.16379
-      }
+      },
+      "Description": "Image to use as a background for the shape.\nDrawn behind the background color."
     },
     {
-      "InputId": "a20d31a3-115f-4d38-a988-2a41f6e5cbea"/*Fill*/,
+      "InputId": "a20d31a3-115f-4d38-a988-2a41f6e5cbea"/*Color*/,
       "Position": {
         "X": -195.9,
         "Y": -83.16377
-      }
+      },
+      "Description": "Fill color of the shape."
     },
     {
       "InputId": "d4c20bfc-3943-4c5d-a8f8-5175fa152e91"/*Background*/,
       "Position": {
         "X": -195.9,
         "Y": 6.8362274
-      }
+      },
+      "Description": "Background color of the shape."
     },
     {
-      "InputId": "ee2f51f9-38bb-4613-b234-173218cb3aae"/*Center*/,
+      "InputId": "ee2f51f9-38bb-4613-b234-173218cb3aae"/*Position*/,
       "Position": {
         "X": -212.96689,
         "Y": 77.20885
       },
-      "GroupTitle": "Position"
+      "GroupTitle": "Position",
+      "Description": "Center position."
     },
     {
       "InputId": "c608e4c4-c545-45ed-8e37-95a53f6871d0"/*Stretch*/,
       "Position": {
         "X": -212.96689,
         "Y": 122.20885
-      }
+      },
+      "Description": "Stretches the shape non-uniformly."
     },
     {
       "InputId": "d2401798-593b-4cc3-85d8-3570ddec1f2a"/*Scale*/,
       "Position": {
         "X": -212.96689,
         "Y": 167.20885
       },
+      "Description": "Scales the shape uniformly.",
```

Comment:
be consistant w/ line 44 : use "Stretch" here.

Reply:
I've used "stretches" for nonuniform scaling and "scales" for uniform scaling, to match the parameters' names. If it would be better to use the same word throughout, I'll change both to "scales".